### PR TITLE
Change to fablabesds

### PR DESCRIPTION
**Pull request by FelixINX**
Hi here is my change for my wiki.

- VisualEditor
- ContactPage
- Feedback

Also how can we change the default language of the wiki? I accidently choose frc instead of fr.

Thanks!

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -649,6 +649,7 @@
 	'wmgUseArticleFeedbackv5' => array(
 		'default' => false,
 		'applewikiwiki' => true,
+		'fablabesds' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Wikiname + wiki

File: LocalSettings.php [hunk 2]
```diff
@@ -740,6 +741,7 @@
 		'default' => false, // Add wiki config to ContactPage.php
 		'ayrshirewiki' => true,
 		'christipediawiki' => true,
+		'fablabesds' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Wikiname + wiki

File: LocalSettings.php [hunk 3]
```diff
@@ -1345,6 +1347,7 @@
 		'evawiki' => true,
 		'extloadwiki' => true,
 		'ezdmfwiki' => true,
+		'fablabesds' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Wikiname + wiki

File: LocalSettings.php (hunk 1 above)
Comment (FelixINX) [inline LocalSettings.php]: @LegoFan4000 please elaborate. I am missing something?

**Comment by FelixINX** [comment]
Thanks @LegoFan4000 . Any idea for the language?

**Comment by LegoFan4000** [comment]
That is done on meta by an admin. As I am not an admin I cannot do that.

**Comment by FelixINX** [comment]
Thanks I'll post an issue on phabricator

**Comment by Reception123** [comment]
@LegoFan4000 It is not done by an admin but by a wiki creator or a system administrator. The administrator permission on Meta has nothing to do with global requests like this.
@FelixINX The rest is good now, except the "No reply user" which I'm not currently sure of.

File: ContactPage.php [hunk 4]
```diff
@@ -42,3 +42,23 @@
 		'RLStyleModules' => array(),
 	);
 }
+
+if ( $wgDBname == 'fablabesdswiki' ) {
+	$wgContactConfig['default'] = array(
+		'RecipientUser' => 'Contact COOP FabLab ESDS',
+		'SenderEmail' => $wgPasswordSender,
+		'SenderName' => 'Ne pas répondre FabLab ESDS',
```
Comment (Reception123) [inline ContactPage.php]: I think this has to be "Miraheze No Reply" but I'm not sure.

Comment (JohnFLewis) [inline ContactPage.php]: It implies a 'Do Not Reply' therefore it is fine, it's only a conical name and has no impact.

Comment (FelixINX) [inline ContactPage.php]: @JohnFLewis Do I need To change it?

**Comment by Reception123** [comment]
@FelixINX Leogfan forgot to mention that parsoid needs to be enabled for VE.

**Comment by LegoFan4000** [comment]
Yes to do that, add ypur wiki[here](https://github.com/miraheze/puppet/blob/master/modules/parsoid/manifests/init.pp).

**Comment by FelixINX** [comment]
Sorry about the delay, now done.

@Reception123 Should the `SenderName` be change?

**Comment by Reception123** [comment]
@FelixINX As JohnLewis said, the SenderName is fine.

**Comment by FelixINX** [comment]
Ok thanks.

**Pull request merged**